### PR TITLE
general improvement

### PR DESCRIPTION
**Pull request by jbruechert**
Sorry for not splitting up patches ...

### CODE REVIEW

File: halium-install [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```
Comment (lnjX) [inline halium-install]: For what is that?

File: functions/post-inst.sh [hunk 2]
```diff
@@ -5,10 +5,16 @@
 #
 # License: GPLv3
 
+do_until_success() {
+	while ! "$@"; do
+		echo "Failed, trying again"
```
Comment (lnjX) [inline functions/post-inst.sh]: Maybe change to "Failed, please try again"

File: halium-install (hunk 1 above)
Comment (jbruechert) [reply halium-install]: It makes the script not continue if one step failed.

Comment (lnjX) [reply halium-install]: Better remove this, it fails for me ...

**Pull request merged**